.lintrunner.toml

```
@@ -96,6 +96,7 @@ include_patterns = [
     'test/test_type_hints.py',
     'test/test_type_info.py',
     'test/test_utils.py',
+    'github/.scripts/*.py',
```

Maybe future proof it via:

```suggestion
    'github/.scripts/**/*.py',
```